--- a/Core/Config.cpp	2026-03-03 19:47:39.586059378 -0400
+++ b/Core/Config.cpp	2026-03-03 19:56:35.498970876 -0400
@@ -1617,10 +1617,10 @@ void Config::PostLoadCleanup() {
 	}
 
 	// Automatically silence secondary instances. Could be an option I guess, but meh.
-	if (PPSSPP_ID > 1) {
-		NOTICE_LOG(Log::Audio, "Secondary instance %d - silencing audio", (int)PPSSPP_ID);
-		g_Config.iGameVolume = 0;
-	}
+	// if (PPSSPP_ID > 1) {
+	// 	NOTICE_LOG(Log::Audio, "Secondary instance %d - silencing audio", (int)PPSSPP_ID);
+	// 	g_Config.iGameVolume = 0;
+	// }
 
 	// Automatically switch away from deprecated setting value.
 	if (iTexScalingLevel <= 0) {
